CONTRIBUTING.md

```diff
@@ -95,5 +95,17 @@ When submitting a PR, please keep the following guidelines in mind:
 
 For an example of a properly formatted PR, take a look at https://github.com/wildfly-security/wildfly-elytron/pull/1532
 
+## Code Reviews
+
+All submissions, including submissions by project members, need to be reviewed by at least two WildFly Elytron committers before being merged.
+
+The [GitHub Pull Request Review Process](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) is followed for every pull request.
+
+## Maintenance Branches
+
+If you are working on a fix that's required for a maintenance branch (e.g., a fix for the 1.15.x or 2.2.x branches), please submit
+your PR directly against the relevant maintenance branch. Once approved and merged, the fix will then be forward ported to the
```

Comment:
I am not sure in this if it is so much "forward ported" that sounds like the work still exists just in a different direction - maybe mention that we merge from maintenance branches to upstream to ensure changes are kept in sync?

Reply:
Thanks, I've updated this.